site/training/training.qmd

```diff
@@ -1,7 +1,8 @@
 ---
-pagetitle: "ValidMind Academy"
+pagetitle: "{{< var vm.product >}} Academy"
```

Comment:
I'd perhaps omit the variable for the academy name or create a new one just for "ValidMind Academy". I think it's possible that we will use a different name at some point and I'd treat the name of our training program as a proper noun in its own right.

Reply:
Another variable to solve the issue:

https://github.com/validmind/documentation/blob/a9a2406774a5c791d6e55128e864fb501ff52e58/site/_variables.yml#L23